qml/pages/ChatPage.qml

```diff
@@ -549,16 +552,17 @@ Page {
                                     id: messageBackground
                                     anchors {
                                         left: parent.left
-                                        leftMargin: (chatPage.myUserId === display.sender_user_id) ? 2 * Theme.horizontalPageMargin : 0
+                                        leftMargin: messageListItem.isOwnMessage ? 2 * Theme.horizontalPageMargin : 0
                                         right: parent.right
-                                        rightMargin: (chatPage.myUserId === display.sender_user_id) ? 0 : 2 * Theme.horizontalPageMargin
+                                        rightMargin: messageListItem.isOwnMessage ? 0 : 2 * Theme.horizontalPageMargin
                                         verticalCenter: parent.verticalCenter
                                     }
                                     height: messageTextColumn.height + ( 2 * Theme.paddingMedium )
 
                                     color: index > ( chatView.count - chatInformation.unread_count - 1 ) ? Theme.secondaryHighlightColor : Theme.secondaryColor
                                     radius: parent.width / 50
                                     opacity: index > ( chatView.count - chatInformation.unread_count - 1 ) ? 0.5 : 0.2
+                                    visible: !messageListItem.isSticker
```

Comment:
Telegram's own [Stickers Done Right](https://telegram.org/blog/stickers) page shows sticker without background bubble as an example. Does that convince you or do you still want to go against the rules and have an option for that?

Reply:
Yep, this an example how the native Telegram client does it. However, I wouldn't derive a "rule" out of it. I simply would like to do it as I wish, especially as te stickers themselves are displayed properly (with transparency etc.).

Reply:
It's a rule in a sense that it's how the UX was designed in the first place, it's the way everyone is doing it (I'm not aware of any exceptions) and that's what users expect to see. But of course rules are to be broken sometimes. I don't have any problem with making it configurable. What do you want to be the default - the way people expect it or the opposite?

My guess is that you probably want to surprise people. If they just see what they expect to see, most of them will never touch that option and your surprise will go unnoticed.

Reply:
I don't want to surprise people. When I started this project I actually wanted to learn how TDLib works and if it's worth contributing to the existing Telegram SFOS projects or build a new one. I wasn't sure if I make this one public for quite a while to be honest. But well, that's probably another story. ;)

Concerning the implementation: It should probably go into the direction of #23. People should be able to enable the background for everything or nothing - and if you want there can be an option to hide it only for stickers.

Reply:
Ok, configurable it will be then. Be prepared for settings to be ripped off `TDLibWrapper` to become a separate object. All that will probably have to wait until next weekend, though.

Reply:
No hurry - and thanks already for the extraction of the app settings! :)